File: sentimental-mario-less/mario.py
import sys
import cs50

while True:
    n = cs50.get_int("Height: ")
    if n > 0 and n <= 8:
        break
# space+# equal to n
for i in range(n):
    for j in range(n-i-1):
        print(" ", end="")
    for k in range(i+1):
        print("#", end="")
    print("")

File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void){
    int n = get_int("Height: ");
    while(n<1){
        n = get_int("Height: ");
    }
    for(int i=0;i<n;i++){
        for(int j=n-i;j>1;j--){
            printf(" ");
        }
        for(int k=0;k<=i;k++){
            printf("#");
        }
        printf("  ");
        for(int k=0;k<=i;k++){
            printf("#");
        }
        printf("\n");
    }

}

